django/chat/templates/chat/components/chat_message.html

```diff
@@ -63,10 +63,14 @@
 
 <script>
 (function () {
-  let element = document.querySelector('#message_{{message.id}} .message-text');
-  if (element && (element.offsetHeight < element.scrollHeight)) {
-    element.closest('.message-outer').classList.add('truncate');
+  function checkTruncation() {
+    let element = document.querySelector('#message_{{message.id}} .message-text');
+    if (element && (element.offsetHeight < element.scrollHeight)) {
+      element.closest('.message-outer').classList.add('truncate');
+    }
   }
+  checkTruncation();
+  setTimeout(checkTruncation, 100);
```

Comment:
Needed to run this code again after a slight delay due to the message needing time to render before its height can be measured accurately. (This is to display the "show more" button when message is truncated)